lib/features/rules/BpmnRules.js

```diff
@@ -1164,10 +1154,18 @@ function canCopy(elements, element) {
     return true;
   }
 
-  if (is(element, 'bpmn:Lane') && !includes(elements, element.parent)) {
```

Comment:
We're getting rid of null-safety here. Is this intentional?

Reply:
Let me have a look.